src/tests/integration/test-data/show/AssertTest.checkFail_assert_false().expected

```diff
@@ -715,14 +715,13 @@ module SUMMARY-TEST%ASSERTTEST.CHECKFAIL-ASSERT-FALSE():0
        andBool ( 0 <=Int TIMESTAMP_CELL:Int
        andBool ( CALLER_ID:Int =/=Int 645326474426547203313410069153905908525362434349
        andBool ( ORIGIN_ID:Int =/=Int 645326474426547203313410069153905908525362434349
-       andBool ( _CONTRACT_ID =/=Int 645326474426547203313410069153905908525362434349
```

Comment:
Do we know why these are disappearing?

Reply:
I added the line
```
 new_init_cterm = new_init_cterm.remove_useless_constraints()
 ```
at the end of `update_node_from_cterm` to get rid of constraints that referred to variables that no longer exist after replacing the contents of certain cells as that function does. I'm pretty sure it comes from that.
I wanted to check with you to make sure that makes sense to do though.

Reply:
Yes, this does make sense, and this constraint is not useful since this is not a CSE test, I believe.